org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/ConsistencyCheckErrors.java

```diff
@@ -0,0 +1,63 @@
+package org.eclipse.winery.repository;
+
+import org.eclipse.winery.common.ids.definitions.DefinitionsChildId;
+
+public class ConsistencyCheckErrors {
+
+	private DefinitionsChildId checkedId;
+	private DefinitionsChildId inconsistentId;
+
+	private boolean biggerError;
+	
+	private String biggerErrorMessage = "";
+	
+	public ConsistencyCheckErrors() {
+		biggerError = false;
+	}
+
+	@Override
+	public String toString() {
+		if (this.isBiggerError()) {
```

Comment:
This smells like a bad practice. Please make an abstract class ConsistencyCheckError with two concrete subclasses `BigConsistencyError` and `SmallConsistencyError`. This will make the boolean variable obsolete.